src/pages/tutor-profile/TutorProfile.jsx

```diff
@@ -54,7 +62,13 @@ const TutorProfile = () => {
       <ProfileInfo myRole={userRole} userData={response} />
       <CompleteProfileBlock data={response} profileItems={profileItems} />
       <AboutTutorBlock />
-      {shouldShowPresentation && <VideoPresentation />}
+      {shouldShowPresentation && (
```

Comment:
for good practice it should be in the new component and use here. Can do that or pass it into variable and render `{showPresentation}` smth like that

Reply:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/101183408/431a639f-4503-4290-a600-d96109311d3a)
fixed